src/app.ts

```diff
@@ -23,6 +25,29 @@ app.use(cors())
 app.use(morgan(":method :url :status - :response-time ms"))
 app.use(express.urlencoded({ extended: true }))
 
+
+//clean all jobs
+async function cleanAllQueues(): Promise<any> {
+  await SchedulerQueue.clean(0, 'delayed');
+  await SchedulerQueue.clean(0, 'wait');
+  await   SchedulerQueue.clean(0, 'active');
+  await  SchedulerQueue.clean(0, 'completed');
+  await  SchedulerQueue.clean(0, 'failed');
+  let multi_1 = SchedulerQueue.multi();
+  await   multi_1.del(SchedulerQueue.toKey('repeat'));
+  await  multi_1.exec();
+
+  await SchedulerReferenceQueue.clean(0, 'delayed');
+  await SchedulerReferenceQueue.clean(0, 'wait');
+  await   SchedulerReferenceQueue.clean(0, 'active');
+  await  SchedulerReferenceQueue.clean(0, 'completed');
+  await  SchedulerReferenceQueue.clean(0, 'failed');
+  let multi_2 = SchedulerReferenceQueue.multi();    
+  await  multi_2.exec();
+
+  await SchedulerQueue.empty()
+  await SchedulerReferenceQueue.empty()
+}
```

Comment:
Can you move this to a utils file and import it?

Reply:
Moved to utils/ActivitySchedulerJob